docs/reference/oras-auth-provider.md

```diff
@@ -175,20 +198,49 @@ Ratify only supports the kubernetes.io/dockerconfigjson secret type or the legac
     }
 ```
 
-Note: Kubernetes secrets are reloaded and refreshed for Ratify to use every 12 hours. Changes to the Secret may not be reflected immediately. 
+Note: Kubernetes secrets are reloaded and refreshed for Ratify to use every 12 hours. Changes to the Secret may not be reflected immediately.
+
+- Create a k8s secret by providing credentials on the command line. This secret should be in the same namespace that contains Ratify deployment.
+
+```bash
+kubectl create secret docker-registry ratify-regcred --docker-server=<your-registry-server> --docker-username=<your-name> --docker-password=<your-pword> --docker-email=<your-email>
+```
+
+- Deploy Ratify using helm
+
+```bash
+helm install ratify charts/ratify --set registryCredsSecret=ratify-regcred
+```
+
+> This mode can be used to authenticate with a single registry. If authentication to multiple registries is needed, docker config file can be used as described below
+
+If Docker config file is used for the registry login process, the same file can be used to create a k8s secret.
+
+- Deploy Ratify by specifying the path to the Docker config file.
+
+> Note: If you use a Docker credentials store, you won't see that auth entry but a credsStore entry with the name of the store as value. In such cases, this option cannot be used. 
+
+```bash
+helm install ratify charts/ratify --set-file dockerConfig=<path to the local Docker config file>
```

Comment:
Actually I think this should be dockerConfig=<docker config file content>. https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#inspecting-the-secret-regcred

Reply:
Not sure what you meant here.  That it should actually be `dockerConfig=.` ?

Reply:
It should be the contents of the docker config file and not the path

Reply:
Looking at the helm chart, this is actually correct since you do this in the dockconfigsecret.yaml file:
`.dockerconfigjson: {{ .Values.dockerConfig | b64enc | quote }}`

Reply:
oh yup you're right. It's a `--set-file` so this works. Thanks for the catch!